test/client/spec/components/victory-axis/victory-axis.spec.js

```diff
@@ -30,6 +32,50 @@ describe("components/victory-axis", () => {
         `0 0 ${VictoryAxis.defaultProps.width} ${VictoryAxis.defaultProps.height}`;
       expect(svg.prop("viewBox")).to.equal(viewBoxValue);
     });
+
+    it("renders 6 ticks", () => {
```

Comment:
This test may pass unreliably because ticks generated from the `tickCount`prop use d3-scale.ticks under the hood https://github.com/d3/d3/wiki/Quantitative-Scales#linear_ticks